api/src/test/java/org/openmrs/api/db/hibernate/HibernateDiagnosisDAOTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.api.db.hibernate;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.openmrs.Diagnosis;
+import org.openmrs.api.db.DiagnosisDAO;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class HibernateDiagnosisDAOTest extends BaseModuleContextSensitiveTest {
+	
+	private static final String DIAGNOSIS_XML = "HibernateDiagnosisDAOTest-Dataset.xml";
```

Comment:
This file will not be found. It needs the full path as here https://github.com/openmrs/openmrs-core/pull/2590/commits/d4981cd723b8dd55064c1b58ae6b0bcc4a19665f#diff-0d16b7abc7a5851414804ac8d1b34bb9R31

Reply:
Done